parser_library/src/processing/statement_processors/ordinary_processor.cpp

```diff
@@ -257,11 +254,15 @@ void ordinary_processor::check_postponed_statements(
         switch (rs->opcode_ref().type)
         {
             case hlasm_plugin::parser_library::context::instruction_type::MACH:
-                low_language_processor::check(*rs, stmt->location_stack(), dep_solver, mach_checker, *this);
+                mach_proc_.check(*rs,
```

Comment:
who calls the `transform_mnemonic` and `transform_default` now?

Reply:
It was still being called by the call to `mach_proc_.check()` which was only implemented in the base class (`low_language_processor`). Nevertheless, it is now reverted back.